public/assets/js/register.js

```diff
@@ -0,0 +1,72 @@
+
+$(document).ready(function () {
+
+    $("#username").keydown(function () {
+      return /[a-z]/i.test(event.key);
+    });
+
+});
+
+function validateEmail() {
+  var email = document.getElementById('email').value;
+  var regex = /^([a-zA-Z0-9_\.\-\+])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+  if(email.length <1){
+    document.getElementById('error').innerHTML = "";
```

Comment:
Didn't I mention this when discussing Abhishek's code that we should not be setting HTML in the function itself.
The function should simply return if email is valid or not.

Reply:
Fixed